tests/e2e/integration/aex2-popups.js

```diff
@@ -55,7 +55,7 @@ describe('Tests cases for AEX-2 popups', () => {
     popups.forEach((popup) => {
       cy.openAex2Popup(popup, popup === 'sign' && SCHEMA.TX_TYPE.spend)
         .get('[data-cy=deny]')
-        .click()
+        .click({ force: true })
```

Comment:
Why `{ force: true }` is needed?

Reply:
Cypress is claiming that the buttons are hidden, but they are visible. By using `{force: true }` we are telling cypress not to check for this.

Reply:
I know what for the `force` option for. The question is what is covering/why is force needed.

Reply:
This is related to the duplicate `#app` issue. After fixing that it now works without the `force` option